style.css

```diff
@@ -0,0 +1,115 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@400;500;700&family=Roboto:wght@400;500;700&display=swap');
+
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style: none;
+  text-decoration: none;
+}
+
+.container {
+  max-width: 375px;
```

Comment:
- [OPTIONAL] I would recommend you to be aware of the use of fixed widths and not use them always when defining the width of sections of a page, because the app can be loaded on different screens with different sizes and for example, if a user with a mobile phone larger than 375px wide opens your app, there is whitespace around the `.container` section you created. So maybe in this case would be better to leave the `width: 100%` and use fixed widths to define the elements inside this section, like the texts.